packages/client/src/client.js

```diff
@@ -129,6 +129,11 @@ class Client {
       });
     });
 
+    // Throw and error incase function not passed
+    if (cb && typeof(cb) !== 'function') {
+      return new Error('Callback passed is not a function.');
```

Comment:
You have to throw the error, e.g. `throw new Error(....)`

Reply:
This was a silly mistake..